### PR TITLE
[BUGFIX] Getting presence on login no longer causes sign-out

### PR DESCRIPTION
**Pull request by plasne**
Fixes:

image

### CODE REVIEW

**Review by TechnicallyWilliams — APPROVED** [review]
This is good.

File: packages/providers/mgt-msal2-provider/src/Msal2Provider.ts [hunk 1]
```diff
@@ -761,6 +762,9 @@ export class Msal2Provider extends IProvider {
             throw popUpErr;
           }
         }
+      } else if (e instanceof BrowserAuthError && e.message.indexOf('no_network_connectivity') > -1) {
```
Comment (MattBillFred) [inline packages/providers/mgt-msal2-provider/src/Msal2Provider.ts:765]: I updated my MGT test application to use a `Login` with a presence:
```
<Login showPresence={true} />
```
Then I quickly reset the network.

I'm seeing a different message as being the cause of the sign out:
image

Comment (plasne) [reply packages/providers/mgt-msal2-provider/src/Msal2Provider.ts:765]: I can add that one as well.

**Review by andrewDoing — APPROVED** [review]

File: packages/providers/mgt-msal2-provider/src/Msal2Provider.ts (hunk 1 above)
Comment (MattBillFred) [reply packages/providers/mgt-msal2-provider/src/Msal2Provider.ts:765]: I wonder if that error is coming from `ChatList`. The new code is stopping signing out, but ChatList gets stuck at the "loading chats" spinner.

Comment (plasne) [reply packages/providers/mgt-msal2-provider/src/Msal2Provider.ts:765]: Loading chats retries 3 times. Are you seeing those messages in the logs?

image

**Review by MattBillFred — APPROVED** [review]

**Pull request merged**